recsys/recsys_tutorial.py

```diff
@@ -300,3 +303,10 @@ def get_data_tensors(df):
 # %%
 test_data_tensors = get_data_tensors(data_test)
 model.evaluate(test_data_tensors[:-1], test_data_tensors[-1], steps=30)
+
+# %% [markdown]
+# ## Summary
+#
+# In this tutorial, we showed one way to use Snorkel for Recommendations. We used books metadata and review text to create `LF`s that estimate user ratings. We used a `LabelModel` to combine the outputs of those `LF`s. Finally, we trained a model to predict what books a user will read and like (and therefore what books should be recommended to the user) based only on what books the user has interacted with in the past.
+#
+# Note that we just focused on a specific setting in this tutorial; but Snorkel can also be used in different settings, where we have user profile data, where users have rated every book they read, and so on.
```

Comment:
Here we demonstrated one way to use Snorkel for training a recommender system. Note, however, that this approach could be easily adapted to take advantage of additional information as it is available (e.g., in the presence of user profile data, with denser user ratings, etc.).